### ISSUE TITLE
Some action options are not automatically selected so you can navigate between them using e.g. arrow keys

### ISSUE THREAD

**Issue by Vulp0d** [CleverRaven/Cataclysm-DDA#78233]
### Describe the bug

I'm not sure if I'm describing this correctly, in previous versions if you were to, for example, butcher a corpse while some enemy was moving towards you, you would get a message about that in which you could either press letters corresponding to an action or use arrow keys to select a desired option and then confirm it. All the time, the first option was always selected. It seems to no longer be the case, at least with ImGui windows.

### Attach save file

N/A

### Steps to reproduce

1. Create a word, player and start a game.
2. Kill something
3. Try to butcher the corpse while an enemy is approaching you.
4. Watch as you cannot select any option using arrow keys, as none is pre-selected

### Expected behavior

The first option to be automatically selected, like in previous versions.

### Screenshots

![bugreport](https://github.com/user-attachments/assets/c0cd2b82-5895-4f67-a4b3-963700244c2d)

### Versions and configuration

- OS: Linux
    - OS Version: LSB Version: n/a; Distributor ID: EndeavourOS; Description: EndeavourOS Linux; Release: rolling; Codename: rolling; 
- Game Version: cdda-experimental-2024-11-29-0753 91e1ec8 [64-bit]
- Graphics Version: Tiles
- Game Language: English [en]
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Portal Storms Ignore NPCs [personal_portal_storms],
    Slowdown Fungal Growth [no_fungal_growth],
    Mind Over Matter [mindovermatter],
    Bombastic Perks [bombastic_perks],
    Martial Mastery [perk_melee_system],
    Translate Complex Dialogue [translate_dialogue],
    SpeedyDex [speedydex],
    Stats Through Kills [stats_through_kills],
    Tamable Wildlife [Tamable_Wildlife]
]

### Additional context

N/A

**Comment by Terrorforge** [comment]
I've noticed that you _can_ select options with the arrow keys (or numpad keys in my case), but it's obviously hard to make sure you select the right thing because you can't see the selector.

**Comment by mosaicex** [comment]
I believe the options are still automatically selected, they just aren't highlighted for some reason.

**Comment by Jlamb96** [comment]
Yup noticed the issue today after hopping back on after about a week break, the actions are selected and change with the arrow keys, but are indeed not highlighted anymore making it much more difficult to accurately pick the correct option.
